src/components/ProfileDropdown.jsx

```diff
@@ -66,66 +65,79 @@ const useStyles = makeStyles((theme) => ({
 export default function ProfileDropdown(props) {
   const classes = useStyles();
   const { signOut, name, email, photo } = props;
+
   const [dropMenu, setDropMenu] = React.useState(null);
+  const handleClick = (event) => setDropMenu(event.currentTarget);
+  const handleClose = () => setDropMenu(null);
 
-  const handleClick = (event) => {
-    setDropMenu(event.currentTarget);
-  };
+  // needs to be updated once there is a working manage profile page
+  const manageProfileLink = '/home';
+
+  function createMenuItem(text, icon, task) {
+    return (
+      <>
+        <MenuItem onClick={task} className={classes.menuListItem}>
+          <ListItemIcon>{icon}</ListItemIcon>
+          <ListItemText primary={text} />
+        </MenuItem>
+      </>
+    );
+  }
 
-  const handleClose = () => {
-    setDropMenu(null);
+  const display = {
+    0: {
```

Comment:
Why create an object/array that you don't iterate through? The `0` key is completely unnecessary.

Reply:
Yeah Initially I planned to use an array to iterate but notice there is only 1 key which is unnecessary, I removed it and changed it to be conditionally rendered with ``&&``